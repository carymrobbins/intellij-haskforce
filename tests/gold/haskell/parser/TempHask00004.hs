{-# LANGUAGE QuasiQuotes #-}
module TempHask00004 where

mkYesod "HelloWorld" [parseRoutes|
/ HomeR GET
|]
