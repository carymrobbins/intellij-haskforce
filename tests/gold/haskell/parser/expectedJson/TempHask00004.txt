ERROR:TemplateHaskell is not enabled