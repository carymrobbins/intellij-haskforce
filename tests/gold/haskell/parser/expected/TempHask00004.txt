Haskell File
  PPRAGMA
    PsiElement(HaskellTokenType.{-#)('{-#')
    PsiElement(HaskellTokenType.PRAGMA)('LANGUAGE')
    PsiElement(HaskellTokenType.PRAGMA)('QuasiQuotes')
    PsiElement(HaskellTokenType.#-})('#-}')
  MODULEDECL
    PsiElement(HaskellTokenType.module)('module')
    QCONID
      CONID
        PsiElement(HaskellTokenType.conidRegexp)('TempHask00004')
    PsiElement(HaskellTokenType.where)('where')
  BODY
    QVAR
      QVARID
        VARID
          PsiElement(HaskellTokenType.varidRegexp)('mkYesod')
    PSTRINGTOKEN
      PsiElement(HaskellTokenType.")('"')
      PsiElement(HaskellTokenType.STRINGTOKEN)('HelloWorld')
      PsiElement(HaskellTokenType.")('"')
    PsiElement(HaskellTokenType.Quasi-[)('[')
    QVARID
      VARID
        PsiElement(HaskellTokenType.varidRegexp)('parseRoutes')
    PsiElement(HaskellTokenType.|)('|')
    QQBLOB
      PsiElement(HaskellTokenType.Quasi-stuff)('\n/ HomeR GET\n')
    PsiElement(HaskellTokenType.|])('|]')